client/src/pages/Signup.tsx

```diff
@@ -1,12 +1,15 @@
 import React, { useState, FormEvent, ChangeEvent } from "react";
 import { useNavigate, Link } from "react-router-dom";
 import { UserAuth } from "../context/AuthContext";
-import "./Signup.css";
 import Input from "../components/Input";
 import Button from "../components/Button";
 import Header from "../components/Header";
 import Footer from "../components/Footer";
 
+import "../styles/index.css";
+import "../styles/App.css";
+import "../styles/Input.css";
+
```

Comment:
These imports are not needed.
- `index.css` is imported in `index.tsx`
- `App.css` is imported in `App.tsx`
- `Input.css` is imported in `Input.tsx`